### PR TITLE
remove extra lines

### PR DESCRIPTION
**Pull request by fanatid**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,10 +6,6 @@
 // Merged Buffer refactorings from base58-native by Stephen Pair
 // Copyright (c) 2013 BitPay Inc
 
-/**
- * @param {string} ALPHABET
- * @return {encode: function, decode: function}
- */
```
Comment (dcousens) [inline index.js]: ping @jprichardson were these needed for anything?

File: index.js [hunk 2]
```diff
@@ -72,11 +56,7 @@ module.exports = function base (ALPHABET) {
       var value = ALPHABET_MAP[string[i]]
       if (value === undefined) throw new Error('Non-base' + BASE + ' character')
 
-      var carry = bytes[0] * BASE + value
-      bytes[0] = carry & 0xff
-      carry >>= 8
```
Comment (dcousens) [inline index.js]: This looks good,  I hadn't tested it but I always thought it wasn't necessary.

File: index.js (hunk 1 above)
Comment (jprichardson) [inline index.js]: Nope, I'm a JSDoc hater.

**Comment by jprichardson** [comment]
@dcousens feel free to publish.

**Comment by dcousens** [comment]
Looks like they were added by @fanatid anyway: https://github.com/cryptocoinjs/base-x/commit/733eead8e16101e125a2687d408ad963769bf32b

**Comment by dcousens** [comment]
Published as `1.0.4` (https://github.com/cryptocoinjs/base-x/commit/fd05cce3a060b617d61991b521783a98c9524c09)

**Comment by fanatid** [comment]
Yep. I added JSDoc :(, now I don't think that was good idea.

**Pull request merged**